src/main/java/mate.academy/service/impl/FileReaderServiceImpl.java

```diff
@@ -4,8 +4,10 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.service.FileReaderService;
 
+@Component
```

Comment:
I think every interface should be marked(but not a class), because if you will make new class and implement this interface you will need to write this annotation again. So as for me better to write Component for interface

Reply:
it's opposite, we mark impls of interface with this annotation to mark what instances should be crated